target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -165,6 +165,11 @@ wn3000rpv3)
 	ucidef_set_led_default "power_r" "POWER (red)" "$board:red:power" "0"
 	set_wifi_led "$board:green:router"
 	;;
+ex3700)
+	ucidef_set_led_default "power_a" "POWER (amber)" "$board:amber:power" "0"
```

Comment:
> The blinking amber led on bootup is used by the stock firmware on this device, so we wanted to replicate this in LEDE. I did a similar thing in the EX2700 port.

We had similar discussion about LEDs in one of recent PR: https://github.com/lede-project/source/pull/865#discussion_r102938528.

If this device contains two LEDs which can be used as status LED, I also prefer to select the green one over the red/amber/orange.